packages/lwc-language-server/src/metadata-utils/content-assets-util.ts

```diff
@@ -20,7 +20,7 @@ function getResourceName(resourceMetaFile: string) {
     return parse(resourceFile).name;
 }
 
-export async function updateContentAssetIndex(updatedFiles: FileEvent[], { workspaceRoot }: WorkspaceContext, writeConfigs: boolean = true) {
+export async function updateContentAssetIndex(updatedFiles: FileEvent[], { workspaceRoots }: WorkspaceContext, writeConfigs: boolean = true) {
```

Comment:
All this wouldn't work in a multi root scenario but thats probably OK. I think we just need to document which methods are used explicitly for SFDX projects vs which ones are used for core / internal development